frontends/web/src/routes/buy/info.tsx

```diff
@@ -65,23 +66,90 @@ class BuyInfo extends Component<Props, State> {
+                                                <col width="50px" />
+                                                <col width="*" />
+                                            </colgroup>
+                                            <thead>
+                                                <tr>
+                                                    <th>{t('buy.info.disclaimer.payment.table.method')}</th>
+                                                    <th>{t('buy.info.disclaimer.payment.table.fee')}</th>
+                                                    <th>{t('buy.info.disclaimer.payment.table.description')}</th>
+                                                </tr>
+                                            </thead>
+                                            <tbody>
+                                                <tr>
+                                                    <td>{t('buy.info.disclaimer.payment.table.1_method')}</td>
+                                                    <td class={style.nowrap}>1.9 %</td>
+                                                    <td>{t('buy.info.disclaimer.payment.table.1_description')}</td>
+                                                </tr>
+                                                <tr>
+                                                    <td>{t('buy.info.disclaimer.payment.table.2_method')}</td>
+                                                    <td class={style.nowrap}>5.9 %</td>
+                                                    <td>{t('buy.info.disclaimer.payment.table.2_description')}</td>
+                                                </tr>
+                                                <tr>
+                                                    <td>{t('buy.info.disclaimer.payment.table.3_method')}</td>
+                                                    <td class={style.nowrap}>5.9 %</td>
+                                                    <td>{t('buy.info.disclaimer.payment.table.3_description')}</td>
+                                                </tr>
+                                            </tbody>
+                                        </table>
+                                    </div>
+                                    <p>{t('buy.info.disclaimer.payment.footnote')}</p>
+                                    <h2 class={style.title}>
+                                        {t('buy.info.disclaimer.security.title')}
+                                    </h2>
+                                    <p>{t('buy.info.disclaimer.security.description', { name })}</p>
+                                    <p>
+                                        <A class={style.link} href="https://shiftcrypto.ch/bitbox02/threat-model/">
+                                            {t('buy.info.disclaimer.security.link')}
+                                        </A>
+                                    </p>
+                                    <h2 class={style.title}>
```

Comment:
I noticed that the sidebar guide talks about Safello, and the link there was changed to `/exchanges`?!

Anyway, what about adding the security and privacy sections to the sidebar guide, have it in locize like a guide entry, but then also show it here in the disclaimer (using the same locize keys of course)? 

and also remove the Safello entry and replace it with a moonpay entry `Please direct your support requests to Moonpay <link>`?

Reply:
updated the guide entry with security and privacy and list of exchanges